### PR TITLE
[Merged by Bors] - chore(Order/*): move `SupSet`, `Set.sUnion` etc to a new file

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/SetNotation.lean [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+- `Set.iUnion s`, `Set.iInter s`: same as `iSup s` and `iInf s`,
+  but works only for indexed families of sets.
+
+## Notation
+
+- `⨆ i, f i`, `⨅ i, f i`: supremum and infimum of an indexed family, respectively;
+- `⋃₀ s`, `⋂₀ s`: union and intersection of a set of sets;
+- `⋃ i, s i`, `⋂ i, s i`: union and intersection of an indexed family of sets.
+
+-/
```
Comment (adomani) [inline Mathlib/Order/SetNotation.lean:32]: Where did this module doc come from?  Shouldn't you be one of the authors?  :)

**Review by adomani — APPROVED** [review]
I checked that no declaration was lost in the move and the split looks very reasonable to me.

File: Mathlib/Order/SetNotation.lean [hunk 2]
```diff
@@ -0,0 +1,302 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Patrick Massot
```
Comment (urkud) [inline Mathlib/Order/SetNotation.lean:4]:
```suggestion
Authors: Johannes Hölzl, Patrick Massot, Yury Kudryashov
```

File: Mathlib/Order/SetNotation.lean (hunk 1 above)
Comment (urkud) [reply Mathlib/Order/SetNotation.lean]: Done.

**Comment by adomani** [comment]
Thanks!

maintainer merge

**Comment by urkud** [comment]
> Thanks!
> 
> maintainer merge

@adomani It doesn't work with a whitespace at the end

maintainer merge

**Pull request closed without merging**